cmd/placement/README.md

```diff
@@ -0,0 +1,54 @@
+This is the kcp-scheduling-placement controller separated from kcp-dev/kcp.
+The purpose of this work is documented in the Summary of [this PR](https://github.com/kcp-dev/edge-mc/pull/58).
+
+#### Run the controller
+Two steps should be taken to run the controller.
+
+First, disable the kcp-scheduling-placement and kcp-workload-placement controllers in kcp.
+There are many ways to disable them. One way is to insert
+```go
+	if true {
+		fmt.Printf("%v controller skipped\n", ControllerName)
+		return nil
+	}
+```
+into pkg/reconciler/scheduling/placement/placement_reconcile.go.reconcile(),
+and insert
+```go
+	if true {
+		fmt.Printf("%v controller skipped\n", ControllerName)
+		return false, nil
+	}
+```
+into pkg/reconciler/workload/placement/placement_reconcile.go.reconcile().
```

Comment:
Add "along with the required import statement".

Reply:
Fixed in a force push. Thanks @MikeSpreitzer !